src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const COST_PER_DAY = 40;
+  const DISCOUNT_FOR_WEEK = 50;
+  const DISCOUNT_FOR_THREE_DAYS = 20;
+
+  let totalCost = days * COST_PER_DAY;
+
+  if (days >= 7) {
```

Comment:
Here, 7 is a magic number, which is not understandable for other people. Please, make a specific constant for it.